### PR TITLE
updage-h1-color-to-#fff-on-homepage

### LINKED ISSUE #100: invisible text
#### Current Behavior

The top level heading is not visible on homepage
image

#### Environment

- **Host OS:** Mac Linux Windows
- **Platform:** Docker or Kubernetes
- **Meshery Server Version:** stable-v
- **Meshery Client Version:** stable-v

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by vishalvivekm**
**Notes for Reviewers**

This PR fixes #100
image

**[Signed commits](../blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by vishalvivekm** [comment]
> Your changes made the invisible text visible 🚀

😆 
Thanks Ritik.

File: README.md [hunk 1]
```diff
@@ -17,10 +17,11 @@ The docker image is pre-built with all the website dependencies installed, which
 
 ### With Golang
 
-If your local development environment has a supported version (v1.21.0+) of Golang installed you can run:
+If your local development environment has a supported version (v1.21.0+) of Golang installed, next you'll need to install extended hugo version as it has necessary SCSS/SASS support. Find all the hugo packages here: https://github.com/gohugoio/hugo/releases/tag/v0.120.4
```
Comment (vishalvivekm) [inline README.md:20]: @leecalcote updated this essential step in order to setup and run the site locally using Golang ( based on how I was able to set it up)

Comment (leecalcote) [reply README.md:20]: Roger that.

**Comment by leecalcote** [comment]
#100 is not a bug that affects Safari users.

**Pull request merged**